language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/DefaultValueGenerationUtil.java

```diff
@@ -270,7 +276,10 @@ private static Optional<String> getDefaultValueForType(TypeSymbol bType, boolean
                 break;
             case TABLE:
                 TypeSymbol rowType = ((TableTypeSymbol) rawType).rowTypeParameter();
-                String rowValue = getDefaultValueForTypeDescKind(rowType).orElse("");
+                String rowValue = "";
+                if (rowType.typeKind() != TypeDescKind.TYPE_REFERENCE) {
+                    rowValue = getDefaultValueForTypeDescKind(rowType).orElse("");
```

Comment:
What's this change about? Can you share a sample case

Reply:
Removed that unnecessary if. I here the rowValue always should be an empty.